crates/auth/tests/mod.rs

```diff
@@ -0,0 +1,4 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: Apache-2.0
+mod routes;
+mod tests;
```

Comment:
How about we move contents of `routes.rs` into this file and move all the tests from `tests/tests` to `tests` directory?
It's a bit difficult to navigate a nested structure like this